### DISCUSSION TITLE
Can output from a qmd document be embedded into revealjs output?

### DISCUSSION THREAD

**Discussion by mpf** [quarto-dev/quarto-cli#8400] [Q&A]
### Description

Following the instructions on [embedding output](https://quarto.org/docs/authoring/notebook-embed.html), I have this line in my revealjs-format qmd document:

```
{{< embed /notes/other-note.qmd#fig-name >}}
```
which produces this error
```
WARNING (/Users/mpf/Applications/quarto/share/filters/main.lua:18383) Shortcode 'embed' not found
```
Are embeds permitted in documents meant for revealjs?

**Comment by mcanouil** [comment]
Which version are you using exactly?
Do you have the proper version?

@mpf you are not at your first post, you know the drill 😉
Could you share a small self-contained "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

![A placeholder image](https://placehold.co/600x400.png)

The end.
````
`````

**Comment by mpf** [comment]
Fair enough! And now that I'm trying to reproduce with a MWE, I see different behaviour, which probably means I don't understand something. Using quarto v1.4.542

````qmd
# slide.qmd
---
title: Slide
format: revealjs
---

## First slide

{{< embed note.qmd#fig-plot >}}
````
````qmd
# note.qmd
---
title: Note
format: html
---

```{julia}
#| label: fig-plot
using Plots
plot(sin)
```
````
Previewing `slide.qmd` gives this output, though I expected a figure.


<img width="500" alt="Screenshot 2024-01-23 at 8 31 34 AM" src="https://github.com/quarto-dev/quarto-cli/assets/109942/b39d2870-2324-4e56-b7e1-4f33dad70127">

**Comment by mcanouil** [comment]
I am closing this:
- #8468
